OMCompiler/Compiler/FrontEnd/ModelicaBuiltin.mo

```diff
@@ -1126,6 +1126,8 @@ function loadFile "load file (*.mo) and merge it with the loaded AST."
   input String fileName;
   input String encoding = "UTF-8";
   input Boolean uses = true;
+  input Boolean notify = true "Give a notification of the libraries and versions that were loaded";
+  input Boolean requireExactVersion = false "If the version is required to be exact, if there is a uses Modelica(version=\"3.2\"), Modelica 3.2.1 will not match it.";
```

Comment:
What does `requireExactVersion` mean for loadFile and friends? Don't you already point to the version you wanted to load?

Reply:
Its for the version of the uses annotation library.

Reply:
Then you are changing how `loadModel` works since that was always true. And your commit comment has nothing about this change in behaviour (only notify).

Reply:
That is even how `loadModel` is working. Try the test model in ticket:6133 with `loadModel`. By default it loads the `PowerSystems` library version `1.0.0` and if you specify `requireExactVersion` to true then it skips loading it since the version specified in uses annotation  is `uses(PowerSystems(version = "0.9.0"))`.

So this PR is not changing anything with `loadModel`.